stocks/Authentication/LoginViewController.swift

```diff
@@ -10,11 +10,25 @@ import UIKit
 import FirebaseAuth
 
 class LoginViewController: ViewController {
+    lazy var image: UIImageView = {
+          let tv = UIImageView()
+          tv.image = UIImage(named: "wallpaper_welcome")
+          return tv
+    }()
+    
+    
+    
     lazy var email: UITextField = {
         let tv = UITextField()
         tv.placeholder = "Email"
         tv.delegate = self
+        tv.textAlignment = .center
```

Comment:
add a green line to the bottom of the text fields. the easiest way to do this is to make a UITextField subclass, and add a 1px height view to the text field. change the cursor color to green.